docs/requirements/Features/human-chat-mentions/requirements.md

```
@@ -0,0 +1,284 @@
+{
+  "msgtype": "m.text",
+  "body": "...",
+  "format": "org.matrix.custom.html",
+  "formatted_body": "...",
+  "m.mentions": {
+    "user_ids": ["@user:server"]
+  }
+}
+```
+
+3. When **also** replying (`m.in_reply_to`), **preserve** reply relation; mentions are orthogonal.
```

markdownlint-cli2 (0.22.0)

[warning] Ordered list item prefix
Expected: 1; Actual: 3; Style: 1/2/3

(MD029, ol-prefix)